hugo/content/downloads.md

```diff
@@ -75,20 +77,20 @@ document.getElementById('suggested-download').innerHTML = getPlatformContent(get
 {{< content-layout/download name="macOS <= 10.12 (x64)" href="https://dl.mumble.info/latest/stable/client-macos-hfs+-x64" osclass="mac">}}
 {{< /content-layout/downloads >}}
 
+<!-- The PPA is completely outdated
 {{< content-layout/downloads >}}
 {{< content-layout/download name="Ubuntu" href="https://launchpad.net/~mumble/+archive/release" osclass="ubuntu">}}
 {{< /content-layout/downloads >}}
+-->
 
 {{< content-layout/downloads >}}
 {{< content-layout/download name="Static macOS server (x64)" href="https://dl.mumble.info/latest/stable/server-macos-x64" osclass="mac">}}
-{{< content-layout/download name="Static Linux server (x86)" href="https://dl.mumble.info/latest/stable/server-linux-x86" osclass="linux">}}
```

Comment:
@davidebeatrici do we have to do something special to create this new end point?

Reply:
Nvm, I figured it out myself